experimental/LieAlgebras/docs/src/weyl_groups_experimental.md

```diff
@@ -23,3 +23,22 @@ isomorphism(::Type{FPGroup}, ::WeylGroup)
 PermGroup(::WeylGroup)
 isomorphism(::Type{PermGroup}, ::WeylGroup)
 ```
+
+## Parabolic subgroups
+
+```@docs
+parabolic_subgroup(::WeylGroup, vec::Vector{<:Integer}, ::WeylGroupElem=one(W))
+parabolic_subgroup_with_projection(::WeylGroup, ::Vector{<:Integer}; ::Bool)
```

Comment:
```suggestion
parabolic_subgroup_with_projection(::WeylGroup, ::Vector{<:Integer})
```
the `@docs` macro somehow does not like keyword arguments, but we can instead just leave them out